### DISCUSSION TITLE
Relationship between Reynolds number and mesh number

### DISCUSSION THREAD

**Discussion by wWL2771140158** [xcompact3d/Incompact3d#165] [Q&A] [answered]
Hello, I am using the Cylinder module to calculate the flow around two parallel cylinders. When I take the Reynolds number as 3000, the grid configuration I use can obtain good results. However, when I use the same grid configuration and reduce the Reynolds number to 200, the results will not converge. Does this mean that the smaller the Reynolds number, the more dense grids are required to ensure the convergence of the calculation results? In other words, what is the relationship between Reynolds number and grid configuration? How do you choose the appropriate configuration grid for a specific Reynolds number?
I would appreciate it if you could help me.

**Comment by slaizet** [comment, marked as answer]
Hi,
There is no magic formula between grid resolution and Reynolds number. It depends on the numerical methods, type of mesh, type of simulations (DNS/LES or other) and flow configuration. At low Reynolds numbers you have some extra viscous stability constraints hence why you might need to reduce the time step and/or increase the spatial resolution. 
The only way to figure out what resolution and time step you need is to do some convergence studies.
Best,
Sylvain

**Reply by gauravity30** [reply]
Hi,
Is this also true for higher Re numbers (of order 10^4 or higher). Because I also had to reduce the time-step very low and increase the mesh resolution. Below a certain mesh resolution and above a certain time-step, the simulation is diverging. 
Thanks,
Gaurav

**Reply by slaizet** [reply]
yes this is true for all Reynolds numbers. There is a formula for DNS: N^3 > Re^(9/4) but this formula does not take into account the accuracy of the numerical methods, the type of numerical methods, local mesh refinements, the use of IBM (with extra stability constraint), etc.
Best,
Sylvain

**Reply by gauravity30** [reply]
Thank you for the answer !
So, trial and error with the help of few formulas like the one above, is the only option to find the suitable `dt` and mesh-size.

Thanks,
Gaurav
